### ISSUE TITLE
Book page: Images and captions lack semantic relationship in the markup

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#193]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

Images and their captions currently lack any semantics conveying their relationship. Semantically group images and captions using a `<figure><img><figcaption>` block, or use `aria-describedby` on the image.

### `<figure>` block: Preferred Implementation

```html
<figure>
    <img src="foo.jpg" alt="A generic image of foo.">
    <figcaption>
        An artist's rendering of foo, painted using oil on canvas.
    </figcaption>
</figure>
```

The `<figure><img><figcaption>` as it uses standard HTML markup and relationship is made clear in the document structure. `<figure>` blocks can also be used to associate a caption with any other type of content (like video and audio), and even captioning other text (like code blocks). See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

### Aria-describedby Implementation

```html
<img src="foo.jpg" alt="A generic image of foo." aria-describedby="the-caption">
<span id="the-caption">
    An artist's rendering of foo, painted using oil on canvas.
</span>
```
Semantically not as strong as using a `<figure>` block, but still conveys the relationship between an image and the caption to a screen reader.

**Comment by greatislander** [comment]
Related: https://github.com/pressbooks/pressbooks/issues/267
